homeassistant/components/stream/hls.py

```diff
@@ -148,7 +148,7 @@ def video_codecs(self) -> tuple:
     def container_options(self) -> Callable[[int], dict]:
         """Return Callable which takes a sequence number and returns container options."""
         return lambda sequence: {
-            "movflags": "frag_custom+empty_moov+default_base_moof+skip_sidx+frag_discont",
+            "movflags": "frag_custom+empty_moov+default_base_moof+frag_discont",
```

Comment:
Can we add a comment here why `skip_sidx` is not included?

Reply:
Refering this PR

Reply:
Sure. Should we refer to the PR or to the issue?

Reply:
Added comment